examples/src/main/java/com/influxdb/v3/IOxExample.java

```diff
@@ -85,7 +87,28 @@ public static void main(final String[] args) throws Exception {
                 stream.forEach(row -> System.out.printf("| %-16s | %-18s |%n", row[1], row[2]));
             }
 
-            System.out.printf("-----------------------------------------%n");
+            System.out.printf("-----------------------------------------%n%n");
```

Comment:
Please create separate example for queries into points, something similar as is https://github.com/InfluxCommunity/influxdb3-csharp/tree/main/Examples/Downsampling